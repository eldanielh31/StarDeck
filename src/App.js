import './App.css';
import { Route, Routes, BrowserRouter } from "react-router-dom";
import CreateCard from './Pages/Card/CreateCard';
import CreatePlanet from './Pages/Planet/CreatePlanet';
import Main from './Pages/Principal/Main';

function App() {
  return (
    <BrowserRouter>
      <Routes>
        <Route path='/' element={<Main />}/>
        <Route path="/createcard" element={<CreateCard />} />
        <Route path="/createplanet" element={<CreatePlanet />} />
      </Routes>
    </BrowserRouter>

  );
}

export default App;
