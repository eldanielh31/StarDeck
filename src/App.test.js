import { render } from '@testing-library/react';
import App from './App';


describe('App', () => {
    
  // Prueba de renderizado
  it('Renderiza correctamente', () => {
      const { container } = render(<App />);
      expect(container.firstChild).toMatchSnapshot();
  });

});
