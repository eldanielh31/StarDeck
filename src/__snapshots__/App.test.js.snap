// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renderiza correctamente 1`] = `
<div
  style="color: rgb(0, 0, 0); font-family: 'Trajan Pro', sans- serif; font-size: 16px; line-height: 1.5;"
>
  <header
    class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
    style="background: rgb(0, 0, 0);"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthXl css-19r6kue-MuiContainer-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-regular css-r6ewbb-MuiToolbar-root"
      >
        <a
          class="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap css-1m03wwp-MuiTypography-root"
          href="/"
        >
          Jugar
        </a>
        <div
          class="MuiBox-root css-1t6c9ts"
        >
          <a
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1bl1uc0-MuiButtonBase-root-MuiButton-root"
            href="/createCard"
            tabindex="0"
          >
            Crear Carta
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </a>
          <a
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1bl1uc0-MuiButtonBase-root-MuiButton-root"
            href="/createPlanet"
            tabindex="0"
          >
            Crear Planeta
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </a>
          <a
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1bl1uc0-MuiButtonBase-root-MuiButton-root"
            href="/createDeck"
            tabindex="0"
          >
            Crear Baraja
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </a>
        </div>
      </div>
    </div>
  </header>
  <div />
</div>
`;
